Makefile

```diff
@@ -30,6 +30,7 @@ build: build/lib build/test build/externaltests
 ${TARGETS}: build
 	./node_modules/.bin/rollup --config rollup.config.js --sourcemap --format umd --name weknowhow.expect -o unexpected.js build/lib/index.js
 	sed -e "s/Symbol.iterator in Object/typeof Symbol === 'function' \&\& Symbol.iterator in Object/;" -i unexpected.js
+	sed -e "s/Array\.from/Array.prototype.slice.call/g;" -i unexpected.js
```

Comment:
🥇 LOL

Reply:
:roll_eyes:

Reply:
For the record, I'm not suggesting that we make this change -- it breaks our source maps. I don't want to merge it until this has been sorted out upstream.